jerry-core/parser/js/js-parser-internal.h

```diff
@@ -118,9 +118,9 @@ typedef struct
  * All data allocated by the parser is
  * stored in parser_data_pages in the memory.
  */
-typedef struct parser_mem_page_t
+typedef struct parser_mem_page_s
```

Comment:
I'm Ok with the current form. Just keep that part of the patch that removes the struct tags where not needed.